db/toblock.c

```diff
@@ -5604,7 +5604,8 @@ static int toblock_main_int(struct javasp_trans_state *javasp_trans_handle,
                 if (iq->tranddl) {
                     if (backed_out) {
                         assert(trans == NULL);
-                        bdb_ltran_put_schema_lock(iq->sc_logical_tran);
+                        if (iq->sc_logical_tran)
+                            bdb_ltran_put_schema_lock(iq->sc_logical_tran);
```

Comment:
Under what circumstances is sc_logical_tran NULL? If master swings, I think we should `goto cleanup` instead of adding blkseq.. Also later at line 5627, we are trying to commit sc_logical_tran and if sc_logical_tran is NULL we have a bigger problem.. I dont think we should reach here at all for master swings